projects/demo/src/modules/components/input-time/examples/3/index.ts

```diff
@@ -15,6 +19,7 @@ import {TUI_INPUT_TIME_OPTIONS, tuiCreateTimePeriods} from '@taiga-ui/kit';
             useValue: {
                 icon: 'tuiIconCheckCircleLarge',
                 mode: 'HH:MM:SS',
+                maxValues: {...TUI_INPUT_TIME_DEFAULT_OPTIONS.maxValues, SS: 165},
```

Comment:
Could you create a different `example-4` instead changes here?

and add example for current case https://github.com/Tinkoff/taiga-ui/issues/1229